packages/babylon/src/plugins/flow.js

```diff
@@ -2267,4 +2268,18 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         /* isAsync */ true,
       );
     }
+
+    skipFlowComment(): ?number {
+      const includeComment =
+        this.input.charCodeAt(this.state.pos + 2) === charCodes.colon &&
+        this.input.charCodeAt(this.state.pos + 3) === charCodes.colon &&
+        4; // check for /*::
+      const flowincludeComment =
+        this.input.slice(this.state.pos + 2, 14) === "flow-include" && 14; // check for /*flow-include
+      const annotationComment =
+        this.input.charCodeAt(this.state.pos + 2) === charCodes.colon &&
+        this.input.charCodeAt(this.state.pos + 3) !== charCodes.colon &&
+        2; // check for /*:
```

Comment:
What is this`&& 2` for?

Reply:
for ``/*:``, we have to advance four steps

Reply:
so maybe those should be named better, something like:
```js
const annotationCommentSteps = ch2 === charCodes.colon && ch3 !== charCodes.colon ? 2 : null
```
or maybe even
```js
const COMMENT_ANNOTATION = '*:'
const annotationCommentSteps = ch2 === charCodes.colon && ch3 !== charCodes.colon ? COMMENT_ANNOTATION.length : null
```
I havent done much work on babylon, so probably @nicolo-ribaudo should decide what's gonna be most readable here

Reply:
@Andarist This seems to be a valid point. @nicolo-ribaudo  Your views?